src/plugins/intel_gna/gna_device_interface.hpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <cstdint>
+#include <gna2-inference-api.h>
+
+enum Gna2AccelerationMode;
+class Gna2Model;
+
+enum class GNARequestWaitStatus {
+    kCompleted = 0,  // and removed from GNA library queue
```

Comment:
consider to add 
kNone = 0, // there was no inference request to determine wait status for

Reply:
done